templates/certification/search-results.html

```diff
@@ -51,7 +51,7 @@ <h1>Ubuntu Certified server hardware</h1>
           }}
         </div>
       </div>
-      {% elif form == "Ubuntu Core" %}
+      {% elif category == "Ubuntu Core" %}
```

Comment:
Does this also need to be changed to Device or it for a future PR?

Reply:
Yeah, updated.